frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -77,6 +77,21 @@ local symbol_prefix = {
         mem_usage = "",
         wifi_status = "",
         wifi_status_off = "",
+    },
+    compact_items = {
+        time = nil,
+        pages_left = BD.mirroredUILayout() and "<" or ">",
+        battery = "",
+        -- @translators This is the footer compact item prefix for the number of bookmarks (bookmark count).
+        bookmark_count = C_("FooterCompactItemsPrefix", "BM"),
+        percentage = nil,
+        book_time_to_read = nil,
+        chapter_time_to_read = BD.mirroredUILayout() and "«" or "»",
```

Comment:
It happens you don't need `BD.mirroredUILayout() and ... or ...`, as the chars you chose are mirrorable, and so, mirrored by the FriBidi code.
You can switch the UI language to Arabic to see.
Without BD.mirroredUILayout() , you'll get > and >> mirrored just fine:
![image](https://user-images.githubusercontent.com/24273478/104241955-b5e0ce00-545e-11eb-9428-7c0ec36ca641.png)

But there's an issue, I guess with you removing the spaces :( You don't get the expected RTL order for 1/2, which should look like 2/1:
![image](https://user-images.githubusercontent.com/24273478/104242031-d6108d00-545e-11eb-8cf4-95e1c222f765.png)

Using a zero-width space instead does not seem to solve it :/
`text = text:gsub('%s', '\xE2\x80\x8B') `

Reply:
I switched to a RTL language and see this

<img width="493" alt="Screen Shot 2021-01-11 at 5 04 48 PM" src="https://user-images.githubusercontent.com/46451014/104243828-34be1280-542f-11eb-8866-3d2b2ce015af.png">

The spaces around the slash appear all right.

Reply:
Sorry, is the issue that the page numbering should also be mirrored (showing as 683/-672 683/11)?

Reply:
The space might be, but that's not the issue:
an arabic/RTL reader expects to read page 11 of 683 as : 683/11.

Other white space that should behave:
https://www.compart.com/en/unicode/bidiclass/WS

May be use a "hair space" :) if that's not too large for you: 
`text = text:gsub("%s", "\xE2\x80\x8A")` (please use double quote " instead of single ')

![image](https://user-images.githubusercontent.com/24273478/104244296-656b6f80-5462-11eb-9360-fb4072791a04.png)
![image](https://user-images.githubusercontent.com/24273478/104244259-58e71700-5462-11eb-9e00-0a680cb251b1.png)

Reply:
For differences in space widths: http://jkorpela.fi/chars/spaces.html

Reply:
Wikipedia isn't horrible either. Just search for a term like "hair space" or "thin space" and you'll end up on https://en.wikipedia.org/wiki/Hair_space#Unicode

Reply:
I'm happy with hair spaces :)

After changing the gsub that strips whitespace only, I'm seeing this now: 683/11

Pages remaining is showing the minus sign like this: 683/671-. Should that be raised as a bug?

Reply:
No, I think `683/671-` might be just what RTL readers expect.
https://util.unicode.org/UnicodeJsps/bidi.jsp?a=-123+%2F+456&p=RTL

Reply:
I think so too, particularly if you consider the - to be a symbol.